pandas/tests/groupby/test_grouping.py

```diff
@@ -641,6 +642,27 @@ def test_groupby_level_index_value_all_na(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "df",
+        [
+            pd.DataFrame([[1, 2, 3]], columns=["a", "b", "c"]).set_index("a"),
+            pd.DataFrame([[1, 2, 3], [4, 5, 6]], columns=["a", "b", "c"]).set_index(
+                "a"
+            ),
+            pd.DataFrame(
+                [[1, 2, 3], [4, 5, 6], [7, 8, 9]], columns=["a", "b", "c"]
+            ).set_index("a"),
+        ],
+    )
+    @pytest.mark.parametrize(
```

Comment:
can you use the fixture all_numeric_reductions (you can simply replace method with this)

Reply:
I've just checked, if I remove the second pytest parametrisation and use `all_numeric_reductions` (from `pandas.conftest`) as the second parameter the test will throw an error of `AttributeError: 'DataFrameGroupBy' object has no attribute 'kurt'`.

Reply:
I think can also use `reduction_func` from the groupby contest. Might need to consolidate those fixtures at a later point

Reply:
In that case tests fail for `ngroup` and `nth` reductions: `AssertionError: assert FrozenList(['a']) == FrozenList(['a', 'c'])` and `TypeError: nth() missing 1 required positional argument: 'n'` respectively.

Reply:
So the test in #29124 is already doing this with all of the groupby funcs though with a different intent (duplicate column labels). Maybe after that gets merged we come back to this and see if this makes sense to fit into that test as well via some sort of parametrization; not sure it's worth doing as a one-off